packages/lit-dev-content/site/tutorials/content/carousel/07.md

```diff
@@ -0,0 +1,34 @@
+Now we're ready to position the elements. We use ReactiveElement's `hasUpdated`
+property to avoid animating the first render, and we move only if selection
+has changed (`this.selected !== this.previous`). We go to the right based on
```

Comment:
Clarify "We go to the right"

specifically "go"

Reply:
Reworded